src/java/src/main/java/com/dnanexus/DXSearch.java

```diff
@@ -900,11 +909,11 @@ public int size() {
         /**
          * Iterator implementation for findDataObjects results.
          */
-        private class ResultIterator
+        public class ResultIterator
```

Comment:
Not private, but default-visibility:

```
@VisibleForTesting
class ResultIterator {

...

@VisibleForTesting
int pageNo() {
```

(The VisibleForTesting doesn't actually mean anything to the compiler-- it's just a convention for humans to understand that the visibility of the method/class was intentionally widened to allow tests to peek at the internals.)